core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -275,6 +277,7 @@ object KafkaConfig {
   val LogFlushSchedulerIntervalMsProp = "log.flush.scheduler.interval.ms"
   val LogFlushIntervalMsProp = "log.flush.interval.ms"
   val LogFlushOffsetCheckpointIntervalMsProp = "log.flush.offset.checkpoint.interval.ms"
+  val LogFlushStartOffsetCheckpointIntervalMsProp = "log.flush.start.offset.checkpoint.interval.ms"
```

Comment:
Could we just reuse LogFlushOffsetCheckpointIntervalMsProp instead of introducing a new config?